.github/workflows/tex-tests.yml

```diff
@@ -56,14 +56,14 @@ jobs:
     - name: Set up python for astropy, numpy dev test
       uses: actions/setup-python@v1
       with:
-        python-version: 3.8
+        python-version: '3.10'
```

Comment:
I assume the python version now needs be listed between ' ' (not done in previous version).

Reply:
Only if it ends in `0`.   Then 3.10 is interpreted as 3.1 causing an error.